app/Http/Controllers/NewsController.php

```diff
@@ -168,7 +168,7 @@ public function destroy($id)
         return redirect()->action('NewsController@index');
     }
 
-    public function approve($id, $approve){
+    public function approve($id, $approve) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->